html/views/wb/tcdg2016wb.js

```diff
@@ -91,8 +95,9 @@ function displayPenalty(t, s, p) { // team skater penalty#
 		penaltyBoxd.html("&nbsp;");
 	}
 
-	var cnt = 0; // Change row colors for skaters on 5 or more penalties, or explusion.
-  var fo_exp = ($($('.Team' + t + ' .Skater.Penalty[id=' + s + '] .BoxFO_EXP')[0]).data("id") != null);
+	// Change row colors for skaters on 5 or more penalties, or who have been expelled.
+	var cnt = 0; 
+    var fo_exp = ($($('.Team' + t + ' .Skater.Penalty[id=' + s + '] .BoxFO_EXP')[0]).data("id") != null);
```

Comment:
indentation is off here

Reply:
It's not showing up this way in Eclipse, so I'm a little puzzled.